src/SolverInterface/objectives.jl

```diff
@@ -0,0 +1,81 @@
+"""
+    setobjective!(m::AbstractMathProgModel, N::Int, b, a_varidx, a_coef, Q_vari, Q_varj, Q_coef)
+
+Set the `N`'th objective in the model `m` to be
+```math
+a^Tx + b + \\frac{1}{2}x^TQx
+```
+where ``a`` is a sparse vector specified in tuple form by `a_varidx`, and
+`a_coef`; ``b`` is a scalar; and the symmetric matrix ``Q`` is defined by the
+triplets in `Q_vari`, `Q_varj`, `Q_coef`.
+
+Duplicate indices in either ``A`` or  a ``Q`` matrix are accepted and will be
+summed together. Off-diagonal entries of ``Q`` will be mirrored, so either the
+upper triangular or lower triangular entries of ``Q`` should be provided. If
+entries for both ``(i,j)`` and ``(j,i)`` are provided, these are considered
+duplicate terms. `a_varidx`, `Q_vari`, `Q_varj` should be collections of
+`VariableReference` objects.
+"""
+function setobjective! end
+
+"""
+    modifyobjective!(m::AbstractMathProgModel, i::Int, args...)
+
+Modify elements of the `i`'th objective depending on the
+arguments `args`. The `i`'th objective will have the form:
+```math
+    a_i^T + b_i + \\frac{1}{2}x^TQ_ix
+```
+There are three cases.
+
+# Modify Constant term
+
+    modifyobjective!(m::AbstractMathProgModel, i::Int, b)
+
+Set the constant term of the `i`'th row objective to `b`.
+
+### Examples
```

Comment:
Since this is a subsection under "Modify constant term", it should have two hashes not three.